model/src/main/java/hu/blackbelt/judo/meta/jsl/runtime/TypeInfo.java

```diff
@@ -564,29 +607,33 @@ public static TypeInfo getTargetType(TransferFieldDeclaration transferFieldDecla
 	}
 
 	public static TypeInfo getTargetType(TransferRelationDeclaration transferRelationDeclaration) {
-		if (transferRelationDeclaration == null) {
+		if (transferRelationDeclaration == null ||
+			transferRelationDeclaration.getReferenceType() == null ||
+			transferRelationDeclaration.getReferenceType().getMap() == null ||
+			transferRelationDeclaration.getReferenceType().getMap().getEntity() == null)
+		{
 			return new TypeInfo(BaseType.UNDEFINED, false);
 		}
 
-		EntityDeclaration entityDeclaration = modelExtension.getReferenceType(transferRelationDeclaration).getMap().getEntity();
+		EntityDeclaration entityDeclaration = transferRelationDeclaration.getReferenceType().getMap().getEntity();
 		return new TypeInfo(entityDeclaration, transferRelationDeclaration.isMany(), false);
 	}
 	
-	public static TypeInfo getTargetType(ActorAccessDeclaration access) {
-		if (access == null || modelExtension.getReferenceType(access).getMap() == null) {
-			return new TypeInfo(BaseType.UNDEFINED, false);
-		}
-
-		EntityDeclaration entityDeclaration = modelExtension.getReferenceType(access).getMap().getEntity();
-		return new TypeInfo(entityDeclaration, access.isMany(), false);
-	}
+//	public static TypeInfo getTargetType(ActorAccessDeclaration access) {
+//		if (access == null || modelExtension.getReferenceType(access).getMap() == null) {
+//			return new TypeInfo(BaseType.UNDEFINED, false);
+//		}
+//
+//		EntityDeclaration entityDeclaration = modelExtension.getReferenceType(access).getMap().getEntity();
+//		return new TypeInfo(entityDeclaration, access.isMany(), false);
+//	}
```

Comment:
Is there any intention. with this commen ted out code?

Reply:
removed